app/shared/reservation-form/ReservationForm.js

```diff
@@ -47,13 +47,13 @@ export function validate(values) {
       if (!begin.isBefore(end)) {
         errors.time = 'Alkamisajan on oltava ennen loppumisaikaa';
       } else {
-        const pattern = /^\d\d:[03]0$/;
+        const pattern = /^\d\d:\d[05]$/;
         const areTimesValid = (
           pattern.exec(values.time.begin.time) &&
           pattern.exec(values.time.end.time)
         );
         if (!areTimesValid) {
-          errors.time = 'Ajan on päätyttävä :00 tai :30';
+          errors.time = 'Ajan on päätyttävä :X0 tai :X5';
```

Comment:
If this error message is shown to end-users, I think it is a bit too cryptic 🙂 

I don't have any great suggestions though.

Reply:
I do not disagree, but the problem is, as you put it, that

> I don't have any great suggestions though.

_Minuutin on oltava jaollinen viidellä_ wouldn't be much of a improvement either.